### PR TITLE
Services arcgis_mu, hadoop*, hml_json edited to use hashed data

### PR DESCRIPTION
**Pull request by cuadradek**
- Services arcgis_mu, hadoop_hbase, hadoop_hdfs and hml_json are now
using getHashedHierarchicalData.
- Services arcgis_mu and hml_json were not manually tested.

### CODE REVIEW

**Review by Johaney-s — APPROVED** [review]

File: gen/hml_json [hunk 1]
```diff
@@ -8,44 +8,41 @@ use JSON::XS;
...
-
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-	my @membersData = $rData->getChildElements;
-	foreach my $mData (@membersData) {
-		my %memberAttributes = attributesToHash $mData->getAttributes;
-		$attributesByLogin{$memberAttributes{$A_USER_LOGIN}} = \%memberAttributes;
+my %usersByLogin;
+
+foreach my $resourceId ($data->getResourceIds()) {
```
Comment (balcirakpeter) [inline gen/hml_json:25]: Again `getMemberIdsForFacility`

**Review by balcirakpeter — CHANGES_REQUESTED** [review]
Please update the version also for the script arcgis_mu.
Please check at least if all scripts can be compiled (when you cannot test them) if you haven't done so already.

File: gen/hadoop_hbase [hunk 2]
```diff
@@ -7,27 +7,23 @@ use perunServicesUtils;
...
 
-my $sortingFunction = getAttributeSorting $A_USER_LOGIN, 1;
-
 my @logins;
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-	my @membersData = $rData->getChildElements;
-	for my $memberAttributes (dataToAttributesHashes @membersData) {
-		push(@logins, $memberAttributes->{$A_USER_LOGIN});
+foreach my $resourceId ($data->getResourceIds()) {
```
[the remark this answers is not in the document]
Comment (cuadradek) [reply gen/hadoop_hbase]: Done

File: gen/hadoop_hdfs [hunk 3]
```diff
@@ -7,27 +7,23 @@ use perunServicesUtils;
...
 
-my $sortingFunction = getAttributeSorting $A_USER_LOGIN, 1;
-
 my @logins;
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-	my @membersData = $rData->getChildElements;
-	for my $memberAttributes (dataToAttributesHashes @membersData) {
-		push(@logins, $memberAttributes->{$A_USER_LOGIN});
+foreach my $resourceId ($data->getResourceIds()) {
```
[the remark this answers is not in the document]
Comment (cuadradek) [reply gen/hadoop_hdfs]: Done

File: gen/hml_json (hunk 1 above)
Comment (cuadradek) [reply gen/hml_json]: Done

**Comment by cuadradek** [comment]
All scripts can be compiled

**Review by balcirakpeter — APPROVED** [review]

**Pull request merged**